content/post/career-ladders-part-1.md

```diff
@@ -0,0 +1,54 @@
+---
+author: Caitlin Moorman
+title: "Why You Need A Career Ladder"
+cover: ""
+draft: true
+publishDate: "2019-01-06"
+tags:
+ - analytics
+ - org
+ - teams
+ - management
+---
+
+Happy New Year! In the blank slate of January, many of us are thinking about what's next. Maybe you need a [roap map](https://www.locallyoptimistic.com/post/roadmap/) for the projects your team will tackle this year, or maybe you need a road map for yourself. What should you focus on this year to get yourself to the next level? How do you help your team do the same? A career ladder is one effective tool to help answer those questions - and on top of that, a good ladder can help mitigate bias and eliminate glue work. This post is the first of a three part series on career ladders:
+- Part 1 (this post): Why you need a career ladder
+- Part 2 (coming soon): How to create an effective career ladder
+- Part 3 (coming soon): How to use a career ladder once you've got it, and what to do as an individual contributor if you don't have one.
+
+
+<!--more-->
```

Comment:
@caitlinmoorman I'd recommend making the following change:
```
appy New Year! In the blank slate of January, many of us are thinking about what's next. Maybe you need a [roap map](https://www.locallyoptimistic.com/post/roadmap/) for the projects your team will tackle this year, or maybe you need a road map for yourself. What should you focus on this year to get yourself to the next level? How do you help your team do the same? A career ladder is one effective tool to help answer those questions - and on top of that, a good ladder can help mitigate bias and eliminate glue work.
<!--more-->
This post is the first of a three part series on career ladders:
- Part 1 (this post): Why you need a career ladder
- Part 2 (coming soon): How to create an effective career ladder
- Part 3 (coming soon): How to use a career ladder once you've got it, and what to do as an individual contributor if you don't have one.
```

The summaries on the homepage will remove any line breaks so it'll remove your breaks and bullets. I learned that the hard way haha

Reply:
👍